src/idpf.rs

```diff
@@ -1785,6 +1785,8 @@ mod tests {
         }
     }
 
```

Comment:
@divergentdave Updating this would take some time. I'm happy to if you think this test is useful, but I'm wondering if it's appropriate to remove it. Consider that we'll have to update the hard-coded values every time we rev `VERSION`.

Reply:
I think it can go now, we've got a variety of good Poplar1 test vectors with encoded public shares, and we get expected public share objects from sharding. This was transitionally important when we just had IDPF implemented.